src/plugins/vis_types/vislib/public/vislib/lib/dispatch.js

```diff
@@ -380,13 +380,12 @@ export class Dispatch {
     return function highlight(element) {
       const label = this.getAttribute('data-label');
       if (!label) return;
-      const dimming = 0.5;
       $(element)
         .parent()
         .find('[data-label]')
         .css('opacity', 1) //Opacity 1 is needed to avoid the css application
         .not((els, el) => String($(el).data('label')) === label)
-        .css('opacity', justifyOpacity(dimming));
+        .css('opacity', justifyOpacity(0.5));
```

Comment:
sorry, what I meant is `justifyOpacity` can be removed completely - it's just normalizing the value which isn't necessary anymore.